client/v2/offchain/builder.go

```diff
@@ -0,0 +1,348 @@
+package offchain
```

Comment:
is this specific to offchain, in the near future we will have a tx builder in client v2. It would be odd to have two. Maybe we leave a todo and an issue to remove the custom builder if its not needed in the future

Reply:
make sense. I'll create an issue once I open the PR